docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+```
+
+To show more rows, use the `--limit` flag.
+
+```sh
+dlt dataset duckdb_dataset head items --limit 50
+```
+
+## Adding sources, destinations, and pipelines to your project
+
+Adding a new entity to an existing dlt+ Project is easy. You can add a new entity to your project by running the `dlt <entity_type> <entity_name> add` command. Depending on the entity you are adding, different options are available. To see all options for adding a destination, for example, you can run `dlt destination add --help`. Let's individually add a source, destination, and pipeline to a new project, replicating the default project we created in the previous chapter.
+
+### Create an empty project
+
+Delete all the files in the `tutorial` folder and run the following command to create an empty project:
+
+```sh
+dlt project init
+```
+
+This will create a project without any sources, destinations, datasets, or pipelines; the project will be named after the folder.
+
+### Add all entities
+
+Now we can add all of our entities individually. This way, we can also give them their own names, which will be useful when having multiple destinations of the same type, for example.
+
+Add a source:
+
+```sh
+# add a new arrow source called "my_arrow_source"
+dlt source my_arrow_source add arrow
+```
+
+Add a destination:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+# this will also create a new dataset called "my_duckdb_destination_dataset"
+dlt destination my_duckdb_destination add duckdb
+```
+
+If you want to create a dataset automatically, you can use the `--dataset-name` flag:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+dlt destination my_duckdb_destination add duckdb --dataset-name my_duckdb_destination_dataset
+```
+
+Now we can add a pipeline that uses the source and destination we just added:
+
+```sh
+# add a new pipeline called "my_pipeline" which loads from my_arrow_source and saves to my_duckdb_destination
+# we select the my_duckdb_destination_dataset with the optional flag
+dlt pipeline my_pipeline add my_arrow_source my_duckdb_destination --dataset_name my_duckdb_destination_dataset
+```
+
+### Adding the core source
+
+You can add multiple entities using CLI commands. Let's add another source - this time, a core source such as a
+([REST API](../../dlt-ecosystem/verified-sources/rest_api/index.md), [SQL database](../../dlt-ecosystem/verified-sources/sql_database/index.md), [filesystem](../../dlt-ecosystem/verified-sources/filesystem/index.md)).
+
+Run the following command to add an SQL database source named `sql_db_1`:
+```sh
+# add a new arrow source called "sql_db_1"
+dlt source sql_db_1 add sql_database
+```
+
+This will add the new source to your `dlt.yml` file:
+
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+  sql_db_1:
+    type: sql_database
+```
+
+The corresponding credential placeholders will be added to `.dlt/secrets.toml`, but you can also define them in `dlt.yml`.
```

Comment:
Maybe this is more a future comment but in the tutorial we should really just talk about our recommended option, not different configuration options that are possible

Reply:
Introducing another configuration place is confusing, we should explain to people right at the start, that there is no big difference here.
For example, we explain different config options in existing OSS tutorials

Reply:
I don't think that's good that we do that in OSS either tbh! You _can_ do a lot of things, but what's the best practice? But anyways let's leave it for a future change if we decide to change it or not